cmd/vfkit/main.go

```diff
@@ -157,6 +158,25 @@ func runVFKit(vmConfig *config.VirtualMachine, opts *cmdline.Options) error {
 		}
 		srv.Start()
 	}
+	go func() {
+		<-exitChan
```

Comment:
Might be worth a comment to explain this will get unblocked when SIGKILL/… is received, otherwise this code is not triggered.